From 44cc06949962243ac0d262a9b1c6166a7840a147 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 14 Dec 2013 16:18:23 +0000
Subject: [PATCH] builder: tests: Fix virt-builder --list output.

This fixes commit a27963b67ba1140c29c145a1ce14dbfc05f8d7da.

(cherry picked from commit 3f433d90f208da8e2b7ea397485dfd8d415726a0)
---
 builder/test-virt-builder-list.sh | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/builder/test-virt-builder-list.sh b/builder/test-virt-builder-list.sh
index ff16abe..083c035 100755
--- a/builder/test-virt-builder-list.sh
+++ b/builder/test-virt-builder-list.sh
@@ -29,6 +29,9 @@ short_list=$($VG ./virt-builder --no-check-signature --no-cache --list)
 
 if [ "$short_list" != "phony-debian             Phony Debian
 phony-fedora             Phony Fedora
+phony-fedora-qcow2       Phony Fedora qcow2
+phony-fedora-qcow2-uncompressed Phony Fedora qcow2 uncompressed
+phony-fedora-no-format   Phony Fedora
 phony-ubuntu             Phony Ubuntu
 phony-windows            Phony Windows" ]; then
     echo "$0: unexpected --list output:"
@@ -57,6 +60,30 @@ Notes:
 
 Phony Fedora look-alike used for testing.
 
+os-version:              phony-fedora-qcow2
+Full name:               Phony Fedora qcow2
+Minimum/default size:    1.0G
+
+Notes:
+
+Phony Fedora look-alike used for testing.
+
+os-version:              phony-fedora-qcow2-uncompressed
+Full name:               Phony Fedora qcow2 uncompressed
+Minimum/default size:    1.0G
+
+Notes:
+
+Phony Fedora look-alike used for testing.
+
+os-version:              phony-fedora-no-format
+Full name:               Phony Fedora
+Minimum/default size:    1.0G
+
+Notes:
+
+Phony Fedora look-alike used for testing.
+
 os-version:              phony-ubuntu
 Full name:               Phony Ubuntu
 Minimum/default size:    512.0M
-- 
1.8.4.2

